run-qa-checks

```diff
@@ -2,7 +2,10 @@
 set -e
 
 qa-checks () {
-    openwisp-qa-check --migration-path "./tests/test_project/migrations"
+    openwisp-qa-check \
```

Comment:
@nemesisdesign @atb00ker Its ready for review. We will need to add css and js test for each module like this to run them in CI.

To document about this addition we can mention it here. http://openwisp.io/docs/developer/contributing.html#coding-style-conventions

Reply:
This section of the README needs to be updated: https://github.com/openwisp/openwisp-utils#openwisp-qa-check